net/net_test.go

```diff
@@ -490,11 +492,22 @@ func TestParsePort(t *testing.T) {
 			port:          "65536",
 			expectedError: true,
 		},
+		{
+			name:          "zero port: allowed",
+			port:          "0",
+			allowZero:     true,
+			expectedError: false,
+		},
+		{
+			name:          "zero port: not allowed",
+			port:          "0",
+			expectedError: true,
+		},
 	}
 
 	for _, rt := range tests {
 		t.Run(rt.name, func(t *testing.T) {
-			actualPort, actualError := ParsePort(rt.port)
+			actualPort, actualError := ParsePort(rt.port, allowZero)
```

Comment:
I see this part, but I don'tr see the part that updates the function itself?

Reply:
I committed it but it's gone. I must have done something. Will add again.

Reply:
can you see it now? :)

Reply:
yes